plaso/parsers/selinux.py

```diff
@@ -19,8 +19,8 @@
 
 * http://selinuxproject.org/page/NB_AL
 * http://blog.commandlinekungfu.com/2010/08/episode-106-epoch-fail.html
-* http://www.redhat.com/promo/summit/2010/presentations/
-taste_of_training/Summit_2010_SELinux.pdf
+* http://www.redhat.com/promo/summit/2010/presentations/taste_of_training/
```

Comment:
http://google.github.io/styleguide/pyguide.html says URL shouldn't be wrapped. Do we have a different styleguide here?

Reply:
pylint was complaining about this, let me have a look